examples/nlp/gpt/conf/gpt_dpo.yaml

```
@@ -97,4 +97,8 @@ model:
     index_mapping_dir: null # path to save index mapping .npy files, by default will save in the same location as data_prefix
     data_prefix: null
 
+  overwrite_base_config:
```

Let's add a comment to make it more understandable (please do it in all files -- feel free to rewrite):

```suggestion
  # Which fields from the base model config should be ignored when merging with this config.
  overwrite_base_config:
```